grant.md

```diff
@@ -1,19 +1,165 @@
 ---
 title: GRANT
-toc: true
+toc: false
 ---
 
-## Description
+The `GRANT` statement grants user [privileges](privileges.html) for interacting with specific databases and tables. 
+
+For privileges required by specific statements, see the documentation for the respective [SQL statement](sql-statements.html).
+
+<style>
+div#toc ul {
+    max-width: 50%;
+}
+</style>
+
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/grant.html %}
 
-## Privileges
+## Required Privileges
+
+The user granting privileges must have the `GRANT` privilege on the target databases or tables.  
+
+## Supported Privileges
+
+Users can be granted the following privileges. Some privileges can be granted for databases and tables, while others can be granted only for tables. 
```

Comment:
this is slightly inaccurate. Privileges can be granted on any level but they may not be useful for a given level.
The distinction is important because privileges are inherited, so if I grant `ALL` at the database level, it's still `ALL` at the table level, even though some of the privileges included by `ALL` mean nothing for databases.